src/ai/backend/manager/models/image.py

```diff
@@ -277,6 +277,7 @@ async def from_alias(
         session: AsyncSession,
         alias: str,
         load_aliases=False,
+        load_registry: bool = False,
```

Comment:
Adding values ​​like boolean flags to functions may be simple to implement at the moment, but it reduces code readability and maintainability later. Wouldn't it be difficult to do it in another way?

Reply:
we can add another function like `from_alias_with_registry()` but it will be redundant.
We have added such boolean flags to control ORM options. e.g. load_aliases, load_kernels etc
but this makes it hard to understand what this function and the flag do in the first glance and as you said, it can reduces readability and maintainability.
Could you suggest any good idea about it?

Reply:
How about taking a list of arguments passed to `query.options` and applying them in a for loop?
The `options pattern` is used a lot for this kind of requirement.

Reply:
An `options pattern` is a pattern that passes options behavior, not flag values, in the following way
https://dragondscv.medium.com/functional-options-pattern-42aada423f56

If you write a function in a way that allows you to pass `options` that take a query as an argument, you can work with it unchanged when new options need to be added later.

Can you do it as part of this work? Or will you refactor it next time?

Reply:
I updated this part. `RelationLoadingOption` enum and `_apply_loading_option()` function are added.
Please check if it is proper